README.md

```diff
@@ -6,7 +6,103 @@ Also it implement two ovs modules:
 
-## Build ovs-forwarder container image
+Also it explains how to configure the setup and the VDPA port inside the container.
+
+## Prerequisites
+Make sure you have installed all of the following prerequisites on the host machine:
+  - With connection tracking:
+    - openvswitch-2.14 and above
+    - MLNX_OFED_LINUX-5.2-2.2.0.0 and above
+    - kernel-5.7 and above with connection tracking modules
+
+  - Without connection tracking:
+    - Openvswitch, openvswitch-2.12 and above
+    - MLNX_OFED_LINUX-5.2-2.2.0.0 and above
+
+  - Align the host OFED version with the container OFED version
+
+## Supported Hardware
+Containerized OVS Forwarder has been validated to work with the following Mellanox hardware:
+- ConnectX-5 family adapters => min FW is 16.29.2002
+- ConnectX-6Dx family adapters => min FW is 22.29.2002
+
+## Enable switchdev mode
+Before starting ovs container, make sure to have vfs in switchdev mode and the vfs are binded
+
+- None vf-lag case:
+  - Create vfs on mlnx port
+    ```
+    echo 4 > /sys/class/net/p4p1/device/sriov_numvfs
+    ```
+  - Unbind vfs for mlnx port
+    ```
+    for i in `lspci -D | grep nox | grep Virt| awk '{print $1}'`; do echo $i > /sys/bus/pci/drivers/mlx5_core/unbind; done
+    ```
+  - Move mlnx port to switchdev mode
+    ```
+    /usr/sbin/devlink dev eswitch set pci/0000:03:00.0 mode switchdev
+    ```
+  - Bind vfs for mlnx port
+    ```
+    $ for i in `lspci -D | grep nox | grep Virt| awk '{print $1}'`; do echo $i > /sys/bus/pci/drivers/mlx5_core/bind; done
+    ```
+
+- vf-lag case:
+  - Create vfs on mlnx port
+    ```
+    echo 4 > /sys/class/net/p4p1/device/sriov_numvfs
+    echo 4 > /sys/class/net/p4p2/device/sriov_numvfs
+    ```
+  - Unbind vfs for mlnx port
+    ```
+    for i in `lspci -D | grep nox | grep Virt| awk '{print $1}'`; do echo $i > /sys/bus/pci/drivers/mlx5_core/unbind; done
+    ```
+  - Move mlnx port to switchdev mode
+    ```
+    /usr/sbin/devlink dev eswitch set pci/0000:03:00.0 mode switchdev
+    /usr/sbin/devlink dev eswitch set pci/0000:03:00.1 mode switchdev
+    ```
+  - Create linux bonding
+    ```
+    vim /etc/sysconfig/network-scripts/ifcfg-bond1
+        NAME=bond1
+        DEVICE=bond1
+        BONDING_MASTER=yes
+        TYPE=Bond
+        IPADDR=<Desired IP>
+        NETMASK=<Desired netmask>
+        IPV6ADDR=<Desired IPV6>
+        ONBOOT=no
+        BOOTPROTO=none
+        BONDING_OPTS=<Bonding options>
+        NM_CONTROLLED=no
+
+    vim /etc/sysconfig/network-scripts/ifcfg-slave1
+        NAME=<Desired name>
+        DEVICE=<PF1>
+        TYPE=Ethernet
+        BOOTPROTO=none
+        ONBOOT=no
+        MASTER=bond1
+        SLAVE=yes
+        NM_CONTROLLED=no
+
+    vim /etc/sysconfig/network-scripts/ifcfg-slave2
+        NAME=<Desired name>
+        DEVICE=<PF2>
+        TYPE=Ethernet
+        BOOTPROTO=none
+        ONBOOT=no
+        MASTER=bond1
+        SLAVE=yes
+        NM_CONTROLLED=no
+    ```
```

Comment:
So I think you should add here the ifup for bond interface and also it's better to check that vf-lag configured with
"dmes | grep lag"

Reply:
done

Reply:
I confusing now. change to 

Create linux bonding interface
    ```

Bring up bond interface 
  ```    
       ifup bond1
```

Reply:
done